sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/ReleaseDownloadListener.java

```diff
@@ -61,19 +63,25 @@
 
     @Override
     public void onStart(long enqueueTime) {
+        AppCenterLog.debug(LOG_TAG, String.format(Locale.ENGLISH, "Start download %s (%d) update.",
+                mReleaseDetails.getShortVersion(), mReleaseDetails.getVersion()));
         SharedPreferencesManager.putInt(PREFERENCE_KEY_DOWNLOAD_STATE, DOWNLOAD_STATE_ENQUEUED);
         SharedPreferencesManager.putLong(PREFERENCE_KEY_DOWNLOAD_TIME, enqueueTime);
     }
 
     @Override
+    @WorkerThread
     public boolean onProgress(final long currentSize, final long totalSize) {
-        AppCenterLog.verbose(LOG_TAG, "downloadedBytes=" + currentSize + " totalBytes=" + totalSize);
+        AppCenterLog.verbose(LOG_TAG, String.format(Locale.ENGLISH, "Downloading %s (%d) update: %d KiB / %d KiB",
+                mReleaseDetails.getShortVersion(), mReleaseDetails.getVersion(),
+                (int) (currentSize / KIBIBYTE_IN_BYTES), (int) (totalSize / KIBIBYTE_IN_BYTES)));
```

Comment:
It was not a float before casting to int.
Why casting long to int though?

Reply:
it was because constant is float. we can cast to long here but int is OK for count of KiB

Reply:
Looks like the better question was, why did we define constants to be float since we do only integer division when using them. We never use the reminder anywhere.

Reply:
It looks like this is the question to you 😄 
https://github.com/microsoft/appcenter-sdk-android/commit/b49cd61a46600121cebadd7f2c5961bec2266549#diff-a0f33b89ff8490fc37240ff9b6165e72R129

Reply:
I guess we should change it (both) to `long`

Reply:
Probably an overlook. We can safely change the type then.